core/NameRef.h

```diff
@@ -114,7 +162,7 @@ class NameRef final : private DebugOnlyCheck<NameRefDebugCheck> {
 
     // Returns the `0` NameRef, used to indicate non-existence of a name
     static NameRef noName() {
-        return NameRef(WellKnown{}, 0);
+        return NameRef();
     }
```

Comment:
Any reason to not mark `NameRef()` and this function as `constexpr`?

Reply:
No good reason; I can do that.

Reply:
Done!